docs/cse/get-started-with-cloud-siem/index.md

```diff
@@ -39,5 +33,10 @@ In this section, we'll introduce the following concepts:
   <p>Learn about the contents of the Insights UI in CSE.</p>
   </div>
 </div>
-
+<div className="box smallbox5 card">
+  <div className="container">
+  <a href="/docs/cse/get-started-with-cloud-siem/onboarding-checklist-cse"><img src={useBaseUrl('img/icons/security/compliance.png')} alt="icon" width="40"/><h4>Onboarding Checklist for Cloud SIEM</h4></a>
+  <p>Get up and running quickly with Cloud SIEM Enterprise administrator tasks.</p>
```

Comment:
Since CSE appears prior to this card - to save space, maybe you should abbreviate?
```suggestion
  <a href="/docs/cse/get-started-with-cloud-siem/onboarding-checklist-cse"><img src={useBaseUrl('img/icons/security/compliance.png')} alt="icon" width="40"/><h4>CSE Onboarding Checklist</h4></a>
  <p>Get up and running quickly with CSE administrator tasks.</p>
```

Reply:
Good suggestion, @kimsauce.